src/cat.jl

```diff
@@ -189,13 +191,13 @@ function concat_chunksize(parents)
     newchunks = map(s -> Vector{Union{RegularChunks,IrregularChunks}}(undef, s), size(parents))
     for i in CartesianIndices(parents)
         array = parents[i]
-        ismissing(array) && continue
+        !isa(array,AbstractArray) && continue
```

Comment:
```suggestion
        array isa AbstractArray || continue
```

Maybe easier to read

Reply:
True